pom.xml

```diff
@@ -407,6 +412,15 @@
             <groupId>commons-logging</groupId>
             <artifactId>commons-logging</artifactId>
           </exclusion>
+          <exclusion>
+            <groupId>commons-io</groupId>
+            <artifactId>commons-io</artifactId>
+          </exclusion>
+          <exclusion>
+            <groupId>org.slf4j</groupId>
+            <artifactId>*</artifactId>
```

Comment:
Had no idea you could exclude a whole groupId like this :facepalm: have always added entries separately. Good to know!

Reply:
Until about 3 hours ago, neither did I!